samples/kafka-strimzi-portworx/config/common.yaml

```diff
@@ -0,0 +1,367 @@
+                    replacement: $1
+                    action: replace
+                  - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_port, __meta_kubernetes_pod_container_port_name]
+                    separator: ;
+                    regex: ^(.+;.*)|(;.*metrics)$
+                    replacement: $1
+                    action: keep
+                  - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+                    separator: ;
+                    regex: (.+):(?:\d+);(\d+)
+                    target_label: __address__
+                    replacement: ${1}:${2}
+                    action: replace
+                  - separator: ;
+                    regex: __meta_kubernetes_pod_label_(.+)
+                    replacement: $1
+                    action: labelmap
+                  - source_labels: [__meta_kubernetes_namespace]
+                    separator: ;
+                    regex: (.*)
+                    target_label: namespace
+                    replacement: $1
+                    action: replace
+                  - source_labels: [__meta_kubernetes_pod_name]
+                    separator: ;
+                    regex: (.*)
+                    target_label: kubernetes_pod_name
+                    replacement: $1
+                    action: replace
+                  - source_labels: [__meta_kubernetes_pod_node_name]
+                    separator: ;
+                    regex: (.*)
+                    target_label: node_name
+                    replacement: $1
+                    action: replace
+                  - source_labels: [__meta_kubernetes_pod_host_ip]
+                    separator: ;
+                    regex: (.*)
+                    target_label: node_ip
+                    replacement: $1
+                    action: replace
+
+  confluent-schema-registry:
+    namespace: "kafka"
+    injectNamespace: true
+    config:
+      replicaCount: 3
+      service:
+        labels: {}
+
+      kafkaStore:
+        overrideBootstrapServers: "PLAINTEXT://kcluster-kafka-brokers.kafka:9092"
+
+      kafka:
+        enabled: false
+
+      ingress:
+        enabled: false
+        
+      external:
+        enabled: false
+
+      jmx:
+        enabled: false
+        port: 5555
+
+      prometheus:
+        jmx:
+          enabled: false
+          image: solsson/kafka-prometheus-jmx-exporter@sha256
+          imageTag: 6f82e2b0464f50da8104acd7363fb9b995001ddff77d248379f8788e78946143
+          port: 5556
+          resources:
+            limits:
+              cpu: 100m
+              memory: 128Mi
+            requests:
+              cpu: 100m
+              memory: 128Mi
+
+      secrets: []
+  # portworx:
+  #   namespace: "portworx"
+  #   injectNamespace: true
+  #   config:
+  #     internalKVDB: true
+  #     etcdEndPoint: # Set internal KVDB to false if you wish to use a custom etcdEndpoint
+  #     AKSorEKSInstall: true
```

Comment:
this should be `AKSInstall: true` (https://github.com/portworx/helm/blob/master/charts/portworx/values.yaml#L11)